frontend/src/Editor/CodeBuilder/CodeHinter.jsx

```diff
@@ -115,6 +140,19 @@ export function CodeHinter({
     }
   };
 
+  const focusPreview = () => (isPreviewFocused.current = true);
+  const unFocusPreview = () => (isPreviewFocused.current = false);
+
+  const copyToClipboard = (text) => {
```

Comment:
Can we use the logic for copying that we have here:   https://github.com/ToolJet/ToolJet/blob/c8fbd625021dfe2b6e41731ec92af2bfb501c988/frontend/src/Editor/LeftSidebar/SidebarInspector.jsx#L85 ? It looks more simple.